src/pages/SellProducts.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SellProducts from '../../pages/SellProducts';
```

Comment:
Is this being imported properly? Can't the component be imported with `./SellProducts`?